### PR TITLE
Nextmillenium bid adapter: Collection of statistics data

### PR DESCRIPTION
**Pull request by mhlm**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

File: modules/nextMillenniumBidAdapter.js [hunk 1]
```diff
@@ -328,6 +333,30 @@ function getWurl({auctionId, requestId}) {
...
+}
+
+function getEventHandler(eventName) {
+  return bid => {
+    const bidder = bid.bidder || bid.bidderCode;
+    if (bidder != BIDDER_CODE) return;
+
+    triggerPixel(`${REPORT_ENDPOINT}?event=${eventName}&bidder=${bidder}&source=pbjs`);
+  }
+}
```
Comment (mhlm) [inline modules/nextMillenniumBidAdapter.js:358]: @patmmccann I corrected the code and now all tests pass. I don't know how to test this new functionality, all the added code does is set an event handler for the bidRequested, bidResponse, noBid and bidTimeout events. Internally, it generates a url and sends data to the server via the `utils.triggerPixel` function.
If you still insist on tests, I will try to add them.

Comment (mhlm) [reply modules/nextMillenniumBidAdapter.js:358]: @patmmccann Can you please tell me what you think about it

Comment (patmmccann) [reply modules/nextMillenniumBidAdapter.js:358]: There are no unit tests on this pr, we're hesitant to assign a reviewer before the pr is complete

Comment (patmmccann) [reply modules/nextMillenniumBidAdapter.js:358]: Seeking some feedback from others on how to test, will get back to you

**Comment by mhlm** [comment]
@patmmccann @jsnellbaker I have added tests, you can check the pull request.

**Review by patmmccann — APPROVED** [review]

File: modules/nextMillenniumBidAdapter.js [hunk 2]
```diff
@@ -328,6 +372,29 @@ function getWurl({auctionId, requestId}) {
...
+  };
+
+  return _eventHandler;
+}
+
+function getEventHandler(eventName) {
+  return bid => {
+    const url = spec.getUrlPixelMetric(eventName, bid);
+    if (!url) return;
+    triggerPixel(url);
```
Comment (patmmccann) [inline modules/nextMillenniumBidAdapter.js:394]: please make all these extra network calls configurable by the pub

**Review by patmmccann — CHANGES_REQUESTED** [review]

**Comment by mhlm** [comment]
@patmmccann I have added an option. Look please.

**Comment by mhlm** [comment]
@jsnellbaker Look please this PR.

**Review by jsnellbaker — APPROVED** [review]
Sorry for the delay, I think this looks okay.

**Comment by mhlm** [comment]
@patmmccann You can check again

**Comment by muuki88** [comment]
Docs PR
https://github.com/prebid/prebid.github.io/pull/4197

**Pull request merged**